index.html

```diff
@@ -3236,6 +3252,54 @@ <h2>Contexts</h2>
   </dl>
 </ol>
 
+<p>To <dfn>deserialize a web frame</dfn> by a
+  JSON <a>Object</a> <var>object</var> that
+  <a>represents a web frame</a>:
+
+<ol>
+  <li><p>If <var>object</var> has no <a>own property</a> <a>web frame identifier</a>,
+    return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+
+  <li><p>Let <var>reference</var> be the result of
+    <a data-lt="getting a property">getting</a>
+    the <a>web frame identifier</a> property
+    from <var>object</var>.
+
+  <li><p>Let <var>browsing context</var> be the result
```

Comment:
Why are the steps here implementation defined? 

It seems like this should be something like:
```
Let <var>browsing context</var> be the <a>browsing context</a> whose <a>window handle</a> is <var>reference</var>, or null if no such <a>browsing context</a> exists.
```

Reply:
Actually not sure. I'll correct that. Thanks for the feedback!